Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -44,12 +44,12 @@
     <% if !root_url.start_with?(HOMEBREW_BOTTLE_DEFAULT_DOMAIN) %>
     root_url "<%= root_url %>"
     <% end %>
-    <% if prefix != BottleSpecification::DEFAULT_PREFIX %>
+    <% if ![Homebrew::DEFAULT_PREFIX, "/usr/local"].include?(prefix) %>
```

Comment:
What's the rationale for moving away from `BottleSpecification::DEFAULT_PREFIX` to `Homebrew::DEFAULT_PREFIX`?

Reply:
The intention of https://github.com/Homebrew/brew/pull/4851 was to move from `BottleSpecification::DEFAULT_PREFIX` to `Homebrew::DEFAULT_PREFIX`, but I missed this one.